stringfuzz/transformers/multiply.py

```diff
@@ -25,6 +28,6 @@ def exit_literal(self, literal):
             literal.value = literal.value * self.factor
 
 # public API
-def multiply(ast, factor):
-    transformed = MultiplyTransformer(ast, factor).walk()
+def multiply(ast, factor, re_range_flag):
```

Comment:
Nitpick: maybe rename `re_range_flag` in this file to something more descriptive, like `skip_re_range`?